README.md

```diff
@@ -150,7 +150,7 @@ export default App
 3. Update `yourConfigHere` with your desired configurations.
 
 ## Demo
-There is a demo hosted on [Demo Link] which allows to check _labeling-tool_ package.
+There is a demo hosted on https://epam.github.io/deps-fe-labeling-tool which allows to check _labeling-tool_ package.
```

Comment:
should we update the patch version?

Reply:
if it is already deployed -> then yes, we have to update version + redeploy.

if no -> no need to update version

Reply:
No, package was not deployed yet